src/main/java/org/jvnet/hudson/test/RestartableJenkinsRule.java

```diff
@@ -62,8 +64,25 @@
      */
     public File home;
 
+    /**
+     * TCP/IP port that the server is listening on.
+     */
+    private final int port;
+
     private static final Logger LOGGER = Logger.getLogger(HudsonTestCase.class.getName());
 
+    public RestartableJenkinsRule() {
+        this.port = 0;
+    }
+
+    public RestartableJenkinsRule(boolean reusePort) {
```

Comment:
To avoid constructor arg pollution, maybe this could be a builder-like method `withReusedPort()` or similar?

Reply:
Done.